client/src/config/advancedSearch/inputTypes.ts

```diff
@@ -26,6 +26,10 @@ export const dimensions: Array<string> = [
   'width',
 ]
 
+export const recordTypes: Record<string, Array<string>> = {
+  agent: ['Person', 'Group'],
```

Comment:
I think instead of being an array of strings, this should be an array of objects, with a `label` and `value`. Other record types have a different value than what their called.

For example, when we add a recordType field for Objects, the values needed in the JSON are 'HumanMadeObject' and 'DigitalObject', but the labels in Advanced Search should be value: 'Physical Object' and 'Digital Object'

See https://github.com/project-lux/lux-marklogic/issues/42 for more examples

Reply:
@clarkepeterf This is all set. I also added a conditional to the `validateAdvancedSearch` function. I realized it wasn't returning true when a user had only selected the Person or Group field as their only input.

Reply:
Looks great!